File: DiceGame/DiceGame_Swift1.2/DiceGame_Swift1.2/DiceGame.swift
//
//  DiceGame.swift
//  Dice
//
//  Created by Andreas Bom on 2015-09-02.
//  Copyright (c) 2015 Andreas Bom. All rights reserved.
//

import Foundation

class DiceGame {
    
    var m_d1:Dice
    var m_d2:Dice
    
    
    init(){
        m_d1 = Dice()
        m_d2 = Dice()
    }
    
    func Play()->Bool{
        m_d1.Roll()
        m_d2.Roll()
        return m_d1.GetValue() + m_d2.GetValue() == 7
    }
    
}

File: 2021/DiceGame/MVCDiceGame/src/model/DiceGame.java
package model;
public class DiceGame {
  Die d1;
  Die d2;

  public DiceGame() {
    d1 = new Die();
    d2 = new Die();
  }

  public boolean playGame() {
    d1.roll();
    d2.roll();

    return d1.getValue() + d2.getValue() == 7;
  }
}
